routes/web.php

```diff
@@ -29,6 +29,7 @@
 
     Route::get('tickets/merge', 'TicketsMergeController@index')->name('tickets.merge.index');
     Route::post('tickets/merge', 'TicketsMergeController@store')->name('tickets.merge.store');
+    Route::post('tickets/status', 'TicketsController@updateStatus')->name('tickets.status.update');
```

Comment:
I would create a new controller called TicketStatusController with the method update this, way we folow the 7 REST actions, the route name is ok!

Reply:
good idea. fixed with a new commit